eng/scripts/Build-Local.ps1

```diff
@@ -3,26 +3,31 @@
 
 [CmdletBinding(DefaultParameterSetName='none')]
 param(
-    [switch] $SelfContained,
-    [switch] $ReadyToRun,
+    [switch] $NoSelfContained,
+    [switch] $NoUsePaths,
     [switch] $Trimmed,
-    [switch] $UsePaths,
     [switch] $AllPlatforms,
-    [switch] $VerifyNpx
+    [switch] $VerifyNpx,
+    [string] $NpxArgs = 'tools list'
 )
 
-$RepoRoot = (Resolve-Path "$PSScriptRoot/../..").Path.Replace('\', '/')
+. "$PSScriptRoot/../common/scripts/common.ps1"
+$root = $RepoRoot.Path.Replace('\', '/')
```

Comment:
just a style comment, but I'd probably put this in a getRoot helper

Reply:
$RepoRoot is already a helper from `eng/common/scripts/common.ps1`.
I'm just flipping the slash direction to avoid Join-Path everywhere

Reply:
I've pulled out all of the changes other than the arg passing fix in JS